incl/footer.html

```diff
@@ -11,7 +11,10 @@
   <div id="sidebar">
     <div id="block-aggregator-feed-1" class="clear-block block block-aggregator">
       <h2 class="title">Recent news</h2>
-      <div class="content">
+      <div style="background-color: blanchedalmond; padding-left: 5px;" class="content">
```

Comment:
```suggestion
      <div style="padding-left: 5px;" class="content">
```
Do we really want to add a special background here for the long term? I think it's better to make it like the other ones just below.

Reply:
The way I viewed this was as a temporary announcement. The goal of the background is to catch the eye but to be effective it should be removed after some time.

Reply:
OK, if it's temporary then fine by me. But maybe leave some issue about closing it a month from now.

Reply:
Done.